sway/input/seatop_default.c

```diff
@@ -279,8 +279,12 @@ static void handle_button(struct sway_seat *seat, uint32_t time_msec,
 	}
 
 	// Handle clicking an empty workspace
-	if (node && node->type == N_WORKSPACE) {
-		seat_set_focus(seat, node);
+	if (node && node->type == N_WORKSPACE ) {
+		if ( state == WLR_BUTTON_PRESSED) {
+			seat_set_focus(seat, node);
+		} else {
+			seat_pointer_notify_button(seat, time_msec, button, state);
```

Comment:
Hmm, we should probably call `seat_pointer_notify_button` unconditionally here?

Reply:
In that case I simply removed the return from the function, `handle_button()` will call `seat_pointer_notify_button()` at the very end, and none of the other functions should run on empty workspaces. Pushed the new version.